crates/bevy_pbr/src/deferred/mod.rs

```diff
@@ -255,6 +255,9 @@ impl SpecializedRenderPipeline for DeferredLightingLayout {
         #[cfg(all(feature = "webgl", target_arch = "wasm32", not(feature = "webgpu")))]
         shader_defs.push("WEBGL2".into());
 
+        #[cfg(feature = "spectral_lighting")]
+        shader_defs.push("SPECTRAL_LIGHTING".into());
```

Comment:
Shaderdefs need to be copied to the meshlet material pipeline stuff.

(Note to maintainers: This seems like a frequent problem with people forgetting meshlet stuff, idk how to prevent it other than me watching out for it 😓)